chat-plugins/mafia.js

```diff
@@ -975,6 +975,19 @@ exports.commands = {
 		},
 		leavehelp: [`/mafia leave - Leave the game. Can only be done while signups are open.`],
 
+		playercap: function (target, room, user) {
+			if (!room || !room.game || room.game.gameid !== 'mafia') return this.errorReply(`There is no game of mafia running in this room.`);
+			if (!user.can('mute', null, room) && room.game.hostid !== user.userid) return false;
+			if (room.game.phase !== 'signups') return this.errorReply(`Signups are already closed.`);
+			if (toId(target) === 'none') target = 20;
+			target = parseInt(target);
+			if (!target || target > 20 || target < 2) return this.parse('/help mafia playercount');
+			if (target === room.game.playerCap) return this.errorReply(`Player cap is already set at ${room.game.playerCap}.`);
+			room.game.playerCap = target;
+			room.game.sendRoom(`Player cap has been set to ${room.game.playerCap}`, {declare: true});
```

Comment:
How should we handle setting the cap to a value lower than the current number of players?

Reply:
Hmm... Probably I would just not allow it to be set < PL